txstratum/commons/client.py

```diff
@@ -73,10 +73,13 @@ async def version(self) -> HathorVersion:
             major, minor, patch = ver.split('.')
             return HathorVersion(int(major), int(minor), int(patch))
 
-    async def get_block_template(self) -> BlockTemplate:
+    async def get_block_template(self, address: str) -> BlockTemplate:
         """Return a block template."""
         assert self._session is not None
-        async with self._session.get(self._get_url('get_block_template')) as resp:
+        params = {}
+        if address:
+            params['address'] = address
```

Comment:
Because the address is not "optional" according to the signature, I'd rather not allow it to be empty or alternatively use `address: Optional[str] = None` on the signature and only omit when `address is None`. Either way I'd prefer is `address = ''` is invalid because it's harder to know if the address is intentionally empty or somewhere up the call stack retrieving the address failed and wasn't treated properly (like an empty env var or cli arg).

Reply:
Good point. I'll do it.

Reply:
Fixed. Can you review again, please?

Reply:
You told me it's good in Slack.